packages/request-manager/e2e/interceptor.test.ts

```diff
@@ -145,6 +145,53 @@ describe('Interceptor', () => {
         });
     });
 
+    describe('WebSocket', () => {
+        // 'ws' package is not a part of this project
```

Comment:
If we use it here, I guess it is now part of the project. Why not adding it as dev dependency?

Reply:
sure, why not.
rebased and forcepushed with "ws" dependency